Description: Adds fake classes for unneeded Apple EAWT library. Taken
 from jeuclid package and adapted.
Author: Scott Howard <showard@debian.org>

Index: triplea-1.3.2.2/src/main/java/com/apple/eawt/ApplicationAdapter.java
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ triplea-1.3.2.2/src/main/java/com/apple/eawt/ApplicationAdapter.java	2011-11-10 17:29:17.542394373 -0500
@@ -0,0 +1,36 @@
+/*
+ *                 Sun Public License Notice
+ * 
+ * The contents of this file are subject to the Sun Public License
+ * Version 1.0 (the "License"). You may not use this file except in
+ * compliance with the License. A copy of the License is available at
+ * http://www.sun.com/
+ * 
+ * The Original Code is NetBeans. The Initial Developer of the Original
+ * Code is Sun Microsystems, Inc. Portions Copyright 1997-2004 Sun
+ * Microsystems, Inc. All Rights Reserved.
+ */
+
+/* this is stub code written based on Apple EAWT package javadoc 
+published at
+ * http://developer.apple.com.  It makes compiling code which uses Apple 
+EAWT
+ * on non-Mac platforms possible.  The compiled stub classes should 
+never be
+ * included in the final product.
+ */
+
+package com.apple.eawt;
+
+public class ApplicationAdapter {
+    public void handleAbout(ApplicationEvent event) {}
+    public void handleOpenApplication(ApplicationEvent event) {}
+    public void handleOpenFile(ApplicationEvent event) {}
+    public void handlePreferences(ApplicationEvent event) {}
+    public void handlePrintFile(ApplicationEvent event) {}
+    public void handleQuit(ApplicationEvent event) {}
+    public void handleReOpenApplication(ApplicationEvent event) {} 
+}
+
+
+
Index: triplea-1.3.2.2/src/main/java/com/apple/eawt/ApplicationBeanInfo.java
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ triplea-1.3.2.2/src/main/java/com/apple/eawt/ApplicationBeanInfo.java	2011-11-10 17:29:17.542394373 -0500
@@ -0,0 +1,29 @@
+/*
+ *                 Sun Public License Notice
+ * 
+ * The contents of this file are subject to the Sun Public License
+ * Version 1.0 (the "License"). You may not use this file except in
+ * compliance with the License. A copy of the License is available at
+ * http://www.sun.com/
+ * 
+ * The Original Code is NetBeans. The Initial Developer of the Original
+ * Code is Sun Microsystems, Inc. Portions Copyright 1997-2004 Sun
+ * Microsystems, Inc. All Rights Reserved.
+ */
+
+/* this is stub code written based on Apple EAWT package javadoc 
+published at
+ * http://developer.apple.com.  It makes compiling code which uses Apple 
+EAWT
+ * on non-Mac platforms possible.  The compiled stub classes should 
+never be
+ * included in the final product.
+ */
+
+package com.apple.eawt;
+
+public class ApplicationBeanInfo extends java.beans.SimpleBeanInfo
+{
+    public java.awt.Image getIcon(int iconKind) { return null; }
+}
+
Index: triplea-1.3.2.2/src/main/java/com/apple/eawt/ApplicationEvent.java
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ triplea-1.3.2.2/src/main/java/com/apple/eawt/ApplicationEvent.java	2011-11-10 17:29:17.542394373 -0500
@@ -0,0 +1,34 @@
+/*
+ *                 Sun Public License Notice
+ * 
+ * The contents of this file are subject to the Sun Public License
+ * Version 1.0 (the "License"). You may not use this file except in
+ * compliance with the License. A copy of the License is available at
+ * http://www.sun.com/
+ * 
+ * The Original Code is NetBeans. The Initial Developer of the Original
+ * Code is Sun Microsystems, Inc. Portions Copyright 1997-2004 Sun
+ * Microsystems, Inc. All Rights Reserved.
+ */
+
+/* this is stub code written based on Apple EAWT package javadoc 
+published at
+ * http://developer.apple.com.  It makes compiling code which uses Apple 
+EAWT
+ * on non-Mac platforms possible.  The compiled stub classes should 
+never be
+ * included in the final product.
+ */
+
+package com.apple.eawt;
+
+public class ApplicationEvent extends java.util.EventObject implements 
+java.io.Serializable
+{
+    public ApplicationEvent(Object source) { super(source); }
+    public java.lang.String getFilename() { return null; }
+    public boolean isHandled() { return false; }
+    public void setHandled(boolean state) {}
+}
+
+
Index: triplea-1.3.2.2/src/main/java/com/apple/eawt/Application.java
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ triplea-1.3.2.2/src/main/java/com/apple/eawt/Application.java	2011-11-10 17:29:17.542394373 -0500
@@ -0,0 +1,45 @@
+/*
+ *                 Sun Public License Notice
+ * 
+ * The contents of this file are subject to the Sun Public License
+ * Version 1.0 (the "License"). You may not use this file except in
+ * compliance with the License. A copy of the License is available at
+ * http://www.sun.com/
+ * 
+ * The Original Code is NetBeans. The Initial Developer of the Original
+ * Code is Sun Microsystems, Inc. Portions Copyright 1997-2004 Sun
+ * Microsystems, Inc. All Rights Reserved.
+ */
+
+/* this is stub code written based on Apple EAWT package javadoc 
+published at
+ * http://developer.apple.com.  It makes compiling code which uses Apple 
+EAWT
+ * on non-Mac platforms possible.  The compiled stub classes should 
+never be
+ * included in the final product.
+ */
+
+package com.apple.eawt;
+
+public class Application
+{
+    public Application() { }
+    public void addAboutMenuItem() {}
+    public void addApplicationListener(ApplicationListener listener) {}
+    public void addPreferencesMenuItem() {}
+    public static Application getApplication() { return null; }
+    public boolean getEnabledAboutMenu() { return false; }
+    public boolean getEnabledPreferencesMenu() { return false; }
+    public static java.awt.Point getMouseLocationOnScreen() { return 
+null; }
+    public boolean isAboutMenuItemPresent() { return false; }
+    public boolean isPreferencesMenuItemPresent() { return false; }
+    public void removeAboutMenuItem() {}
+    public void removeApplicationListener(ApplicationListener listener) 
+{}
+    public void removePreferencesMenuItem() {}
+    public void setEnabledAboutMenu(boolean enable) {}
+    public void setEnabledPreferencesMenu(boolean enable) {}
+}
+
Index: triplea-1.3.2.2/src/main/java/com/apple/eawt/ApplicationListener.java
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ triplea-1.3.2.2/src/main/java/com/apple/eawt/ApplicationListener.java	2011-11-10 17:29:17.542394373 -0500
@@ -0,0 +1,35 @@
+/*
+ *                 Sun Public License Notice
+ * 
+ * The contents of this file are subject to the Sun Public License
+ * Version 1.0 (the "License"). You may not use this file except in
+ * compliance with the License. A copy of the License is available at
+ * http://www.sun.com/
+ * 
+ * The Original Code is NetBeans. The Initial Developer of the Original
+ * Code is Sun Microsystems, Inc. Portions Copyright 1997-2004 Sun
+ * Microsystems, Inc. All Rights Reserved.
+ */
+
+/* this is stub code written based on Apple EAWT package javadoc 
+published at
+ * http://developer.apple.com.  It makes compiling code which uses Apple 
+EAWT
+ * on non-Mac platforms possible.  The compiled stub classes should 
+never be
+ * included in the final product.
+ */
+
+package com.apple.eawt;
+
+public interface ApplicationListener extends java.util.EventListener
+{
+    public void handleAbout(ApplicationEvent event);
+    public void handleOpenApplication(ApplicationEvent event);
+    public void handleOpenFile(ApplicationEvent event);
+    public void handlePreferences(ApplicationEvent event);
+    public void handlePrintFile(ApplicationEvent event);
+    public void handleQuit(ApplicationEvent event);
+    public void handleReOpenApplication(ApplicationEvent event);
+}
+
Index: triplea-1.3.2.2/src/main/java/com/apple/eawt/CocoaComponent.java
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ triplea-1.3.2.2/src/main/java/com/apple/eawt/CocoaComponent.java	2011-11-10 17:29:17.542394373 -0500
@@ -0,0 +1,31 @@
+/*
+ *                 Sun Public License Notice
+ * 
+ * The contents of this file are subject to the Sun Public License
+ * Version 1.0 (the "License"). You may not use this file except in
+ * compliance with the License. A copy of the License is available at
+ * http://www.sun.com/
+ * 
+ * The Original Code is NetBeans. The Initial Developer of the Original
+ * Code is Sun Microsystems, Inc. Portions Copyright 1997-2004 Sun
+ * Microsystems, Inc. All Rights Reserved.
+ */
+
+/* this is stub code written based on Apple EAWT package javadoc published at
+ * http://developer.apple.com.  It makes compiling code which uses Apple EAWT
+ * on non-Mac platforms possible.
+ */
+
+
+package com.apple.eawt;
+
+public abstract class CocoaComponent extends java.awt.Canvas
+{
+    public abstract int createNSView();
+    public long createNSViewLong() { return 0; }
+    public abstract java.awt.Dimension getMaximumSize();
+    public abstract java.awt.Dimension getMinimumSize();
+    public abstract java.awt.Dimension getPreferredSize();
+    public void sendMessage(int messageID, java.lang.Object message) {}
+}
+
Index: triplea-1.3.2.2/src/main/java/com/apple/eawt/package
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ triplea-1.3.2.2/src/main/java/com/apple/eawt/package	2011-11-10 17:29:17.542394373 -0500
@@ -0,0 +1,35 @@
+/*
+ *                 Sun Public License Notice
+ * 
+ * The contents of this file are subject to the Sun Public License
+ * Version 1.0 (the "License"). You may not use this file except in
+ * compliance with the License. A copy of the License is available at
+ * http://www.sun.com/
+ * 
+ * The Original Code is NetBeans. The Initial Developer of the Original
+ * Code is Sun Microsystems, Inc. Portions Copyright 1997-2004 Sun
+ * Microsystems, Inc. All Rights Reserved.
+ */
+
+/* this is stub code written based on Apple EAWT package javadoc 
+published at
+ * http://developer.apple.com.  It makes compiling code which uses Apple 
+EAWT
+ * on non-Mac platforms possible.  The compiled stub classes should 
+never be
+ * included in the final product.
+ */
+
+package com.apple.eawt;
+
+public interface ApplicationListener extends java.util.EventListener
+{
+    public void handleAbout(ApplicationEvent event);
+    public void handleOpenApplication(ApplicationEvent event);
+    public void handleOpenFile(ApplicationEvent event);
+    public void handlePreferences(ApplicationEvent event);
+    public void handlePrintFile(ApplicationEvent event);
+    public void handleQuit(ApplicationEvent event);
+    public void handleReOpenApplication(ApplicationEvent event);
+}
+
